src/tests/readytorun/coreroot_determinism/coreroot_determinism.csproj

```diff
@@ -4,8 +4,6 @@
     <CLRTestKind>BuildAndRun</CLRTestKind>
     <CLRTestPriority>0</CLRTestPriority>
     <CLRTestTargetUnsupported Condition="'$(RuntimeFlavor)' != 'coreclr'">true</CLRTestTargetUnsupported>
-    <!-- See https://github.com/dotnet/runtime/issues/50704 -->
-    <GCStressIncompatible>true</GCStressIncompatible>
```

Comment:
This test should pass after #56770 disabled propagating `COMPlus_GCStress` environment variable to `crossgen2`.